<?xml version="1.0"?>
<launch>
  <arg name="num_samples_pos" default="1000" />
  <arg name="num_samples_vel" default="1000" />
  <arg name="chain_start" default="base" />
  <arg name="chain_end" default="right_hand" />
  <arg name="timeout" default="0.005" />
  <arg name="urdf_param" default="/robot_description" />
  <arg name="use_random_position_seed" default="false"/>
  <arg name="use_delta_position_seed" default="false"/>
  <arg name="delta_position_seed_value" default="0.2"/>
  <arg name="nullspace_bias_task" default="false"/>
  <arg name="nullspace_gain" default="0.3"/>

  <node name="sns_ik_tests" pkg="sns_ik_examples" type="all_ik_tests" output="screen">
    <param name="num_samples_pos" value="$(arg num_samples_pos)"/>
    <param name="num_samples_vel" value="$(arg num_samples_vel)"/>
    <param name="chain_start" value="$(arg chain_start)"/>
    <param name="chain_end" value="$(arg chain_end)"/>
    <param name="timeout" value="$(arg timeout)"/>
    <param name="urdf_param" value="$(arg urdf_param)"/>
    <param name="use_random_position_seed" value="$(arg use_random_position_seed)"/>
    <param name="use_delta_position_seed" value="$(arg use_delta_position_seed)"/>
    <param name="delta_position_seed_value" value="$(arg delta_position_seed_value)"/>
    <param name="nullspace_bias_task" value="$(arg nullspace_bias_task)"/>
    <param name="nullspace_gain" value="$(arg nullspace_gain)"/>
  </node>

</launch>
